.prettierrc.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   singleQuote: true,
   printWidth: 120,
   tabWidth: 2,
+  arrowParens: 'avoid',
```

Comment:
?

Reply:
since prettier updated(?) I am getting weird behaviour. Had to add above to stop > 200 errors about lambda single arg parens e.g `before => before  + 2` was now throwing and requiring `(before) => before + 2`